Comparison Engine/Comparison Engine/GoogleMap/MapController.cs

```diff
@@ -24,6 +24,10 @@ private MapController()
         {
         }
 
+        // set default values so it would compile, but will need to set these depending on user or from current location
+        public double prefDistance { private get; set; } = 69; 
+        public string homeAddress { private get; set; } = "S. Nėries 99";
```

Comment:
should probably have a separate variable for current location in the future, not just home address

Reply:
will create a task on trello including this